src/database.py

```diff
@@ -67,8 +67,11 @@ def getObfuscatedId(self):
 		alpha = "0123456789abcdefghijklmnopqrstuv"
 		return ''.join(alpha[n%32] for n in (value, value>>5, value>>10, value>>15))
 	def getObfuscatedKarma(self):
-		offset = round(abs(self.karma * 0.2) + 2)
-		return self.karma + randint(0, offset + 1) - offset
+		if abs(self.karma) > 50:
+			return max(-50, min(self.karma, 50))
+		if abs(self.karma) > 10:
+			return max(-10, min(self.karma, 10))
```

Comment:
one more: is it intentional that bucketing applies to 11 and more or 51 and more?
I'd rather have it be `>=`